action.yml

```
@@ -58,6 +58,13 @@ inputs:
     description: "If true a MySQL 5.7 will be set up"
     required: false
     default: true
+  mysql-schema:
```

I'd go with a generic db-provider here as it will allow us to run different database engines as well, not being forks or feature-matching engines with MySQL.
```suggestion
  db-provider:
```